.eslintrc

```diff
@@ -1,8 +1,9 @@
 {
-    "extends": ["eslint:recommended", "plugin:react/recommended"],
-    "parser": "babel-eslint",
-    "env": {
-        "browser": true,
-        "jest": true
-      }
-}
\ No newline at end of file
+  "extends": ["eslint:recommended", "plugin:react/recommended"],

```

Comment:
I always suggest having separated MR for reformat/prettier and actual feature or bug fix. Because it's hard to review

Reply:
But I started to use promises in this branch, so I added es6 to .eslintrc